spec.html

```diff
@@ -3313,6 +3313,16 @@ <h1>Well-Known Intrinsic Objects</h1>
                 The constructor of async iterator objects (<emu-xref href="#sec-asyncgeneratorfunction-constructor"></emu-xref>)
```

Comment:
This description should match the description you've used for `%GeneratorFunction%`.